File: frontend/src/models/Parameter.ts
export const EVENT_NOTIFICATION_RECIPIENTS_PARAMETER =
  "DESTINATARIOS_NOTIFICACION_EVENTO";

export interface Parameter {
  IdParameter: number;
  nameParameter: string;
  valueParameter: string;
}

export interface ParameterCreate {
  nameParameter: string;
  valueParameter: string;
}

export interface ParameterUpdate {
  valueParameter: string;
}

File: backend/app/domain/entities/Parameter.py
from app.infrastructure.db.connection import Base
from sqlalchemy import Column, Integer, String, Text


class Parameter(Base):
    __tablename__ = "Parameter"

    IdParameter = Column(Integer, primary_key=True, autoincrement=True)
    nameParameter = Column(String(150), nullable=False, unique=True)
    valueParameter = Column(Text, nullable=False)
